[styles] Tweaks for leisure=track

AntonM030481 opened the pull request:
- Added support for[ leisure=track](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dtrack) with area (visible from z15).
![image](https://user-images.githubusercontent.com/85622715/187467092-250ca66f-31bf-45d3-b54a-73e2deb754df.png)

- Linear leisure=track now is visible from z15 (was from z14).
Typically it is used for tracks around pitches and at z14 it is not necessary and often not elegant.
z15:
![image](https://user-images.githubusercontent.com/85622715/187466120-ea9cd92e-d28e-4909-bfad-dc092664eb3b.png)
z14:
![image](https://user-images.githubusercontent.com/85622715/187466702-6fc5298d-7fb9-4b5a-951c-f52acac314b0.png)

biodranik commented:
А некрасивый коричневый цвет нельзя поменять на что-то более приятное и менее выделяющееся? Таким цветом рисуются треки для джипов )

AntonM030481 commented:
Было бы логично использовать спортивный цвет, чтобы площадные и линейные трэки были похожи.

AntonM030481 committed:
[styles] Tweaks for leisure=track

1. Added support for leisure=track with area (visible from z15).
2. Linear leisure=track is visible from z15 (was from z14).
Color changed to @sport.

Signed-off-by: Anton Makouski <anton.makouski@gmail.com>

AntonM030481 commented:
![image](https://user-images.githubusercontent.com/85622715/187911450-0b9d8d89-01ea-4c74-a0e0-b571de8bfb6c.png)

AntonM030481 committed:
[styles] Regenerated

Signed-off-by: Anton Makouski <anton.makouski@gmail.com>

AntonM030481 committed:
Test for leisure-track-area

Signed-off-by: Anton Makouski <anton.makouski@gmail.com>
